demo-ui/test/uk/gov/ons/addressIndex/demoui/controllers/ApplicationHomeTest.scala

```diff
@@ -0,0 +1,25 @@
+package uk.gov.ons.addressIndex.demoui.controllers
+
+import org.scalatestplus.play.PlaySpec
+import play.api.i18n.MessagesApi
+import play.api.mvc.Results
+import play.api.test.Helpers._
+import play.api.test.{FakeRequest, WithApplication}
+import scala.concurrent.ExecutionContext.Implicits.global
+
+class ApplicationHomeTest extends PlaySpec with Results {
+  "Application controller" should {
+    "include at least one link" in new WithApplication {
+      // Given
+      val messagesApi = app.injector.instanceOf[MessagesApi]
+      val expectedString = "Find an address"
+      // When
```

Comment:
@Mironor I'm not up to speed with the Guice injection piece, but do we get to choose between using the Scala implicit API versus the dependency injection framework?